app/org/sagebionetworks/bridge/models/accounts/SignIn.java

```diff
@@ -26,7 +32,7 @@ public String getPassword() {
         return password;
     }
     
-    public boolean isBlank() {
```

Comment:
Not used anywhere in the code. The standard way we validate data from these payload objects is by using a Validator, which allows us to return all the errors keyed in a map a client can use to display errors to a user.